README.md

```diff
@@ -192,7 +196,23 @@ already implemented one that matches your need.
    your TokenizerWrapper subclass. Otherwise, you can directly use TokenizerWrapper.
 
 
-5) **transformers.Pipeline**:
+5) **MetricHandler**:
+    This class is responsible for postprocess operations applied to prediction outputs.
+    In this particular class it is needed to transform the prediction outputs to a
```

Comment:
"In this particular class it is needed to transform" -> "This is performed for transforming"